addons/kaptain/0.x/kaptain-1.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kaptain
+  labels:
+    kubeaddons.mesosphere.io/name: kaptain
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    kudo.kubeaddons.mesosphere.io/create-namespace: "true"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.0.1-1"
+    appversion.kubeaddons.mesosphere.io/kubeflow: "1.0.1"
+spec:
+  namespace: kubeflow
```

Comment:
I suppose we can't rename the namespace, KUDO package reference, etc. without breaking lots of stuff, right?

Reply:
namespace might be a bit tricky, which needs some experiments.

KUDO package reference can be updated, after the kudo-kf updates the operator name from "kubeflow" to kaptain.